### PR TITLE
Fix "-keep-docs" option in ocamlopt manpage

### PR DESCRIPTION
**Pull request by emillon**
It was a duplicate of `-keep-locs` below.

Thanks!

### CODE REVIEW

**Comment by damiendoligez** [comment]
I know credit where credit is due, but do we really need a Changes entry for this? @gasche

**Comment by gasche** [comment]
I would go with whatever the original author felt like.

File: Changes [hunk 1]
```diff
@@ -132,6 +132,9 @@ Working version
 - GPR#1187: Minimal documentation for compiler plugins
   (Florian Angeletti)
 
+- GPR#1220: Fix "-keep-docs" option in ocamlopt manpage
```
Comment (damiendoligez) [inline Changes:135]: I know, credit where credit is due, but do we really need a Changes entry for this? @gasche

**Comment by emillon** [comment]
`CONTRIBUTING.md` insists on that, so I thought that any change was required to have one.

Thanks!

**Pull request merged**